ui/src/config.js

```diff
@@ -32,7 +32,9 @@ export const appConfig = {
   environment: process.env.REACT_APP_ENVIRONMENT || "dev",
   homepage: process.env.REACT_APP_HOMEPAGE || process.env.PUBLIC_URL,
   appIcon: "graphApp",
-  docsUrl: process.env.REACT_APP_USER_DOCS_URL,
+  docsUrl: process.env.REACT_APP_USER_DOCS_URL || [
+    { href: "https://github.com/gojek/turing", label: "Turing User Guide" }
+  ],
```

Comment:
Hmm, shouldn't this be something like:
```js
docsUrl: [
    { 
        href: process.env.REACT_APP_USER_DOCS_URL || "https://github.com/gojek/turing", 
        label: "Turing User Guide" 
    }
  ]
```
?
It's a bit weird to have REACT_APP_USER_DOCS_URL as a JSON array

Reply:
`REACT_APP_USER_DOCS_URL` is a JSON string so we've update `config.js` to use JSON.parse to get docs link value.